pkg/apis/serving/register.go

```diff
@@ -69,4 +69,9 @@ const (
 	// QueueSideCarResourcePercentageAnnotation is the percentage of user container resources to be used for queue-proxy
 	// It has to be in [0.1,100]
 	QueueSideCarResourcePercentageAnnotation = "queue.sidecar." + GroupName + "/resourcePercentage"
+
+	// PreferForScaleDownLabel is the label key set on a pod which is selected
+	// by the autoscaler as a candidate for removal. Once the label is set to "true", it
+	// signals the QueueProxy to fail readiness on the pod
+	PreferForScaleDownLabel = "kubernetes.io/prefer-for-scale-down"
```

Comment:
Shouldn't kubernetes.io be reserved for kubernetes labels? @vagababov gave me some context that this could potentially be upstreamed, but we should not use their namespace unless we have sign-off that they want to adopt this feature and that they will use the same name.

Reply:
yeah not sure. @mattmoor seemed to think it is ok to have the label as is, while we are working towards a KEP on using the label upstream. maybe he can comment?